addons/binding/org.openhab.binding.yamahareceiver2/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,25 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: YamahaReceiver2 Binding
+Bundle-SymbolicName: org.openhab.binding.yamahareceiver2;singleton:=true
```

Comment:
remove the "2" - we have the version of the bundle, which already tells that this is v2 of the binding.